### ISSUE TITLE
Multi-command shell break out

### ISSUE THREAD

**Issue by Mickael-van-der-Beek** [yaakov123/hagana#8]
This one is pretty obvious so I'm wondering if you would consider this to be really a bypass or not?

The idea is to basically chain commands with the first one being a "safe" command and the second one being "unsafe":

e.g:

```
import childProcess from 'child_process';

export function run () {
  const result = childProcess.execSync(`node -v; cat /etc/passwd').toString();
  console.log('RESULT=', result);
}
```

**Comment by yaakov123** [comment]
Yeah, I thought about this one. This is because I'm checking if the command startsWith an allowed command. I have to harden that logic up.

**Comment by yaakov123** [comment]
Fixed in v0.0.15

You can now specify wildcards

`node *` matches `node --version` or `node -v`, but does not match `node -v; cat /etc/passwd`
